test/parser/JavaMetrics.test.ts

```diff
@@ -218,7 +226,7 @@ describe("Java metrics tests.", () => {
             await testFileMetric(
                 javaTestResourcesPath + "/" + FileMetric.complexity + "/SwitchStatement.java",
                 FileMetric.complexity,
-                13
+                8
             );
         });
```

Comment:
Nit: add line break here for consistent style, also for the following "it" calls.